flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlAlterTable.java

```diff
@@ -98,6 +111,15 @@ public LinkedHashMap<String, String> getPartitionKVs() {
         return SqlPartitionUtils.getPartitionKVs(getPartitionSpec());
     }
 
+    /**
+     * Whether to ignore the error if the table doesn't exist.
+     *
+     * @return true when IF EXISTS is specified.
+     */
+    public boolean ignoreIfNotExists() {
```

Comment:
BTW, I think we should keep the code style same. In the `SqlAlterFunction`, we use `isIfExists` as method name.